tests/server/scheduler/policy/test_default_policy.py

```diff
@@ -48,7 +48,7 @@ def setUp(self):
         self.kvs_client = kvs_client.MockAnnaClient()
         self.ip = '127.0.0.1'
 
-        self.policy = DefaultCloudburstSchedulerPolicy(self.pin_socket,
+        self.policy = HeftCloudburstSchedulerPolicy(self.pin_socket,
```

Comment:
Hm. It seems like we would: (1) want to have HEFT specific tests; (2) not want to remove the existing policy tests. At the very minimum, we should duplicate the existing tests to have HEFT versions instead of replacing the existing tests.